projects/angular-formio/src/components/formioreport/formioreport.component.ts

```diff
@@ -17,7 +17,7 @@ export class FormioReportComponent extends FormioComponent implements OnInit, On
   @Output() fetchDataError = new EventEmitter<any>();
   @ViewChild('report', { static: true }) declare formioElement?: ElementRef<any>;
 
-  public isLoading: boolean;
+  // public isLoading: boolean; // Was not allowing yarn build
```

Comment:
I'm not so certain about this change in the context of this ticket - I gather this causes a TypeScript error? If that's the case, how does this build ever succeed? This library is routinely built and published in our build pipelines

Reply:
Overlooked before committing. Retested, looks like it is possibly redundant and can be removed.

![image](https://github.com/formio/angular/assets/112976114/72fd7c82-5f95-480b-97a0-a7ffb08422cb)

Reply:
I guess my issue is that "possibly redundant" is not enough evidence for me to clobber a class instance variable in a PR that is just trying to publish a dev tag to npm. If we're not absolutely certain, let's check with the author of the code (Tanya?), who may say "you can just add an initializer to the class" or "sure go ahead it's not necessary"

Reply:
`FormioReportComponent` extends `FormioComponent` which extends `FormioBaseComponent` which includes an isLoading property already, which I believe it where our redundancy is coming from.

https://github.com/formio/angular/blob/9d4eb9edf903db5d635dd73f53413ab995dc84a4/projects/angular-formio/src/FormioBaseComponent.ts#L71